### DISCUSSION TITLE
What determines if a portrait thumbnail is shown cropped or not?

### DISCUSSION THREAD

**Discussion by bcsteeve** [immich-app/immich#3000] [Q&A] [answered]
![image](https://github.com/immich-app/immich/assets/7412219/375c74c5-aae7-4447-8499-1599384af028)

Here's an example.   The first three are portrait and display as I would expect.  The next two are landscape and display how I would expect.  But the two pictures with the kangaroos are both portrait, and certainly when I click the image it displays correctly, but in the thumbnail it is instead cropped and shown as though it is landscape.

I tried deleting the image and re-uploading and that made no difference, so is there some logic that determines this?

Thanks.

**Comment by brighteyed** [comment]
Could you please post the `exiftool` output for the orientation of your image? I confirm that we incorrectly determine aspect ratio for images with Rotate 270 CW orientation

**Reply by bcsteeve** [reply]
Ah, ok.  So it is based on exif data.  I won't be able to check until after work, but that gives me a clue on how to fix it myself, so thank you.

Is there a plan to accommodate 270CW in a future release?

**Reply by brighteyed** [reply, marked as answer]
The fix has just been released in v1.65.0

**Reply by bcsteeve** [reply]
Ha!  Awesome.  I just saw that in the changelog and tested it and it worked perfectly!
